ci/requirements/min-all-deps.yml

```diff
@@ -7,33 +7,33 @@ dependencies:
   # Run ci/min_deps_check.py to verify that this file respects the policy.
   # When upgrading python, numpy, or pandas, must also change
   # doc/user-guide/installing.rst, doc/user-guide/plotting.rst and setup.py.
-  - python=3.8
+  - python=3.9
   - boto3=1.20
   - bottleneck=1.3
   - cartopy=0.20
   - cdms2=3.1
   - cfgrib=0.9
   - cftime=1.5
   - coveralls
-  - dask-core=2021.11
-  - distributed=2021.11
+  - dask-core=2022.1
+  - distributed=2022.1
   - flox=0.5
-  - h5netcdf=0.11
+  - h5netcdf=0.13
```

Comment:
Great, we can get rid of this decorator then: https://github.com/pydata/xarray/blob/b21f62ee37eea3650a58e9ffa3a7c9f4ae83006b/xarray/tests/__init__.py#L67